docs/users/installation.md

```diff
@@ -157,7 +157,7 @@ Great! You are all set to use Scalafix with sbt :)
 | `scalafixConfig` | `SettingKey[Option[File]]` | `.scalafix.conf` file to specify which scalafix rules should run, together with their potential options. Defaults to `.scalafix.conf` in the root directory, if it exists.
 | `scalafixDependencies` | `SettingKey[Seq[ModuleID]]` | Dependencies making [custom rules](#run-custom-rules) available via their simple name. Can be set in `ThisBuild` or at project-level. Defaults to `Nil`.
 | `scalafixOnCompile` | `SettingKey[Boolean]` | When `true`, Scalafix rule(s) declared in `scalafixConfig` are run on compilation, applying rewrites and failing on lint errors. Defaults to `false`.
-| `scalafixResolvers` | `SettingKey[Seq[Repository]]` | Custom resolvers where `scalafixDependencies` are resolved from. Can be set in `ThisBuild` or at project-level. Defaults to: Ivy2 local, Maven Central, Sonatype releases & Sonatype snapshots.
+| `scalafixResolvers` | `SettingKey[Seq[Repository]]` | Custom resolvers where `scalafixDependencies` are resolved from, in addition to user-defined sbt `resolvers`. Must be set in `ThisBuild`. Defaults to: Ivy2 local, Maven Central, Sonatype releases & Sonatype snapshots.
```

Comment:
https://github.com/scalacenter/sbt-scalafix/pull/404 effectively [removes the ability to define project-level `scalafixResolvers`](https://github.com/scalacenter/sbt-scalafix/pull/404/files#diff-06169c9b17ca1913e8659adfb6e67d1360dcfadd638128457e2d74a9e4dcc72aR440), but it was not working for some cases (with local rules), and it was never requested anyway.